docs(blog): add environment security group rules and ELB access logs (#3888)
aws/copilot-cli: site/content/blogs/release-v121.en.md

## CloudFront Integration

> ## Configure Environment Security Group

Reviewer: What do you think of creating a section like this and moving these under the it?
```
## New Environment Manifest Features

### Configure Environment Security Group
...
### Add Support For ELB Access Logs

```

You can now configure your environment security group rules through environment manifest.   
Sample security group rules template inside environment manifest is given below.
```yaml
network:
  vpc:
    security_group:
      ingress:
        - ip_protocol: tcp
          ports: 0-65535
          cidr: 0.0.0.0/0
      egress:
        - ip_protocol: tcp
          ports: 80
          cidr: 0.0.0.0/0

> ```

Reviewer: Should we add a link to the manifest field here to see the specification?
```
For the complete specification, see the [environment manifest](path/to/link#href).
```

## `job logs`

## Package Addons CloudFormation Templates

## ELB Access Logs Support
You can now enable Elastic Load Balancing access logs that capture detailed information about requests sent to your load balancer.
There are a few ways to enable access logs: 

1. You can specify `access_logs: true` in your environment manifest as shown below and Copilot will create an S3 bucket where the Public Load Balancer will store access logs.
```yaml
name: qa
type: Environment

http:
  public:
    access_logs: true 

> ```

Reviewer: What do you think of saying that they can view the bucket name with `env show --resources`?

Author: sounds good

2. You can also bring in your own bucket and prefix. Copilot will use those bucket details to enable access logs. 
You can do that by specifying the following configuration in your environment manifest.
```yaml
name: qa
type: Environment

http:
 public:
   access_logs:
     bucket_name: my-bucket

>      bucket_prefix: my-prefix

Reviewer: Is it just `prefix` now? 
```suggestion
     prefix: my-prefix
```

Author: Yes, so before making changes here I made sure to add `prefix` in the implementation part so sent out a other PR. Now I will make it `prefix` here.

## CloudFront Integration

## Configure Environment Security Group
You can now configure your environment security group rules through environment manifest. 

Sample security group rules template inside environment manifest is given below.

```yaml
network:
  vpc:
    security_group:
      ingress:
        - ip_protocol: tcp
          ports: 0-65535
          cidr: 0.0.0.0/0
      egress:
        - ip_protocol: tcp
          ports: 80
          cidr: 0.0.0.0/0
```

> [See detailed walkthrough](../docs/manifest/environment.en.md#http-public-access-logs)

Reviewer 2: Access logs aren't the right thing to link to here. Is there a detailed explanation of the security group you meant to link to? Otherwise I think there should be a small explanation here in the blog.

Author: oops I will change this.

for how to specify security group rules to add to the environment security group.
## `job logs`

## Package Addons CloudFormation Templates

## ELB Access Logs Support
You can now enable Elastic Load Balancing access logs that capture detailed information about requests sent to your load balancer. 

> There are couple ways how you can enable access logs: 

Reviewer 2: ```suggestion
There are a few ways to enable access logs: 
```

> 1. You can specify `access_logs: true` in your environment manifest as shown below and Copilot will create an S3 bucket to which the Public Load Balancer will store access logs.

Reviewer 2: ```suggestion
1. You can specify `access_logs: true` in your environment manifest as shown below and Copilot will create an S3 bucket where the Public Load Balancer will store access logs.
```

```yaml
name: qa
type: Environment

http:
  public:
    access_logs: true 
```

2. You can also bring in your own bucket and prefix. Copilot will use those bucket details to enable access logs. 

> You can do that by specifying following configuration in your environment manifest.

Reviewer 2: ```suggestion
You can do that by specifying the following configuration in your environment manifest.
```

```yaml
name: qa
type: Environment

http:
 public:
   access_logs:
     bucket_name: my-bucket
     bucket_prefix: my-prefix
```

> In the option 2 you need to make sure that the bucket exists and has required [bucket policy](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-access-logs.html#attach-bucket-policy) for the load balancer to

Reviewer 2: ```suggestion
When importing your own bucket, you need to make sure that the bucket exists and has the required [bucket policy](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-access-logs.html#attach-bucket-policy) for the load balancer to
```

> write access logs to it.

Reviewer 2: Unnecessary newline here.
